tests/cephadm/test_redeploy_service.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+
+
+class RedeployServiceError(Exception):
+    pass
+
+
+def verify_redeploy_demon(installer_node, redeploy_daemon):
```

Comment:
instead of `installer_node` use `node` and `redeploy_daemon` to `daemon`

Reply:
Updated